site/src/main/java/uk/nhs/digital/ps/beans/navigation/CiBreadcrumbProvider.java

```diff
@@ -0,0 +1,122 @@
+package uk.nhs.digital.ps.beans.navigation;
+
+import org.hippoecm.hst.container.RequestContextProvider;
+import org.hippoecm.hst.content.beans.standard.HippoBean;
+import org.hippoecm.hst.content.beans.standard.HippoFolder;
+import org.hippoecm.hst.core.component.HstRequest;
+import org.hippoecm.hst.core.request.HstRequestContext;
+import org.onehippo.forge.breadcrumb.om.BreadcrumbItem;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.nhs.digital.ps.beans.*;
+
+import java.util.*;
+
+/**
+ * <p>
+ * Default Hippo Breadcrumb plugin uses menu/folder structure to generate breadcrumbs,
+ * but this doesn't fit for clinical indicator content (ciLanding, publication,
+ * series, datasets) as they have a very specific setup in the CMS.
+ * </p>
+ */
+public class CiBreadcrumbProvider  {
+
+    private static final Logger log = LoggerFactory.getLogger(CiBreadcrumbProvider.class);
+    private static final String PATH_CI_LANDING_PAGES = "publication-system/ci-hub/";
+    private static final String SEPARATOR = "/";
+
+    private static final Map<String, String> ciFolderToCiLandingMap = new HashMap<String, String>() {
```

Comment:
I'm not sure I like how hard coded this is. Could we make it so the behaviour is that the CI Folder is the same as the CI Landing page? We map have to rename a few of the landing pages but this shouldn't be two much of a problem if we add a couple of redirects. Then we wouldn't have to change this code every time a landing page changes. See what @ziemsky thinks

Reply:
I agree with @jacsamell that it would be good to not to have this hardcoded.

Following our discussion, let's:
* add an optional field to the `cilanding` doctype, one that would point to the corresponding folders; if Hippo allows liking docs to folders (unlikely) let's use that, if not, let's use a simple text field,
* instead of populating the map with hardcoded values, have a query looking for all documents of type `cilanding` page that have the 'linking field' populated and use those to populate the map,
* after release, establish the cilanding-folder links in production.

Walt and Pete need to be made aware of the function of the linking field (whatever form it takes) and the fact that it'll need populating right after the release.

Reply:
As discussed, updated to make it more dynamic. New field in `ciLanding` added (let me know if better name suggestion for field 😬).